torchci/clickhouse_queries/queue_times_historical/params.json

```diff
@@ -0,0 +1,5 @@
+{
+    "granularity": "String",
+    "startTime": "String",
```

Comment:
I can use DateTime64(3) here directly, i.e. https://github.com/pytorch/test-infra/pull/5572.  Syntax-wise, it looks like a clearer type than the generic String IMO

Reply:
Although I have a question from https://github.com/pytorch/test-infra/pull/5572/files#r1722398187 about ISO time format that somehow ClickHouse fails to process, maybe you know a way to correctly handle them.

Reply:
Yes, `parseDateTimeBestEffort` parses time format with "Z", it's directly compatible with the way we passed dates to Rockset, which simplifies migraition.